cmd/goal/node.go

```diff
@@ -164,9 +167,18 @@ var catchupCmd = &cobra.Command{
 				URL := fmt.Sprintf(catchpointURL, genesis)
 				label, err := getMissingCatchpointLabel(URL)
 				if err != nil {
-					reportErrorf(errorCatchpointLabelMissing, errorUnableToLookupCatchpointLabel)
+					reportErrorf(errorCatchpointLabelMissing, errorUnableToLookupCatchpointLabel, err.Error())
 				}
 				args = append(args, label)
+				if !noPrompt {
```

Comment:
Can we avoid double-negative conditionals, by calling the flag "autoYes" or similar?

Reply:
noPrompt -> fastCatchupForce